qds_sdk/cluster.py

```diff
@@ -204,6 +204,24 @@ def _parse_create_update(cls, args, action):
                                   action="store_true", default=None,
                                   help="Use hbase on this cluster",)
 
+        hbase_group = argparser.add_argument_group("hbase settings")
+        hbase_group.add_argument("--frequency-num",
+                                  dest="frequency_num",
+                                  type=int,
+                                  help="number of times to schedule backup")
+        hbase_group.add_argument("--frequency-unit",
+                                  dest="frequency_unit",
+                                  help="unit of frequency")
+        hbase_group.add_argument("--s3-location",
+                                  dest="s3_location",
+                                  help="location of where to store snapshot")
+        hbase_group.add_argument("--ttl",
```

Comment:
Which ones of these should be required ? IMO - frequency-num, --frequency-unit and s3 location should be required. Will also require tests.